owncloudData/src/main/java/com/owncloud/android/data/spaces/db/SpacesEntity.kt

```diff
@@ -67,6 +67,7 @@ data class SpacesEntity(
     companion object {
         const val DRIVE_TYPE_PERSONAL = "personal"
         const val DRIVE_TYPE_PROJECT = "project"
+        const val DRIVE_ALIAS_SPACES_SHARES = "virtual/shares"
```

Comment:
That was what I thought initially, but the `driveType` is `virtual` and most probably won't be only used by shares, 
 so the actual identifier is the `driveAlias` in this case: `virtual/shares`

Reply:
Apparently, the way to identify the shares drive is by a hardcoded `spaceId` instead of the `driveAlias` or `driveType` at the moment. So I will update it to rely on the hardcoded id for the moment.

Pending on https://github.com/owncloud/ocis/issues/4843